packages/blockchain-wallet-v4-frontend/src/assets/locales/es.json

```diff
@@ -459,6 +459,9 @@
   "copy.event_type": "Evento",
   "copy.events": "Eventos",
   "copy.exchange": "Exchange",
+  "copy.exchange_promo.body": "Comisiones más bajas, más pares para cambiar y soporte directo 24/7 en el Exchange de Blockchain.com",
```

Comment:
I manually added this. Once the release PR get's to dev branch I will have to update lokalise too